examples/svm_batch.py

```diff
@@ -55,11 +55,17 @@ def main(readcsv=read_csv, method='defaultDense'):
     # Prediction result provides prediction
     assert(predict_result.prediction.shape == (pdata.shape[0], 1))
 
-    return (predict_result, plabels)
+    # result of classification
+    decision_result = predict_result.prediction
+    predict_labels = np.where(decision_result >=0, 1, -1)
+
+    return (decision_result, predict_lables, plabels)
```

Comment:
```suggestion
    return (decision_result, predict_labels, plabels)
```

Reply:
Sorry, the variable name has been corrected